scripts/teahouse/send_th_invites.py

```diff
@@ -152,9 +152,11 @@ def inviteGuests(prof, inviter):
         format="%(asctime)s - %(levelname)s - %(message)s",
         level=logging.DEBUG
     )
-    logging.info("Reading configuration file: '{}'.".format(args.config_file))
-    config_reader.load_config_module(args.config_file)
     try:
+        logging.info("Reading configuration file: '{}'.".format(
+            args.config_file)
+        )
+        config_reader.load_config_module(args.config_file)
         send_invitations()
     except:
```

Comment:
[`logging.exception`](https://docs.python.org/2/library/logging.html?highlight=logging#logging.Logger.exception) logs the original exception.